Remotion/Web/Development.WebTesting/Configuration/IWebTestSettings.cs

```diff
@@ -0,0 +1,138 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.DownloadInfrastructure;
+
+namespace Remotion.Web.Development.WebTesting.Configuration
+{
+  /// <summary>
+  /// Represents WebTesting specific settings.
+  /// </summary>
+  public interface IWebTestSettings
+  {
+    /// <summary>
+    /// Browser in which the web tests are run.
+    /// </summary>
+    string BrowserName { get; }
+
+    /// <summary>
+    /// Specifies how long the Coypu engine should maximally search for a web element or try to interact with a web element before it fails.
+    /// </summary>
+    TimeSpan SearchTimeout { get; }
```

Comment:
Defautl please.

Reply:
`SearchTimeout` is a required configuration value and therefore has no default value.